openshift/openfaas-route.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: route.openshift.io/v1
```

Comment:
Would this make sense in the helm chart?

Reply:
Ideally yes. OpenShift does not directly use helm, although it can be added on. Most people I have worked with in OpenShift don't use it either. I can research this more and add it in once I understand it, as I don't think this deployment option would hurt.

Reply:
OK, so perhaps we can make it optional if someone sets "openshift.createRoute: true" in their `values.yaml`?

Reply:
Also we generate YAML from our helm chart. So I think this should be there. I'll merge it now however, with a view to do the above.